src/UserService.Broker/Requests/ProjectService.cs

```diff
@@ -30,10 +30,17 @@ public class ProjectService : IProjectService
       _globalCache = globalCache;
     }
 
-    public async Task<List<ProjectData>> GetProjectsAsync(Guid userId, List<string> errors)
+    public async Task<List<ProjectData>> GetProjectsAsync(Guid userId, List<string> errors, bool includeUsers = true, bool? acsendingSort = true)
```

Comment:
Why we need make acsendingSort  nullable?

Reply:
null in ascending sort means that there isn't any sort:)
false - is descending sort
true - ascending

Reply:
Interesting. But it's not very obvious.